tools/rna_tools/sortmerna/sortmerna.xml

```diff
@@ -15,7 +15,7 @@
     </stdio>
```

Comment:
We should increase the tool version number.
In general every change that affects reproducibility should yield in an increase of the tool version number.